### PR TITLE
Fix difference between GAE_IMAGE_LABEL and actual image tag when released

### PR DESCRIPTION
**Pull request by meltsufin**

### CODE REVIEW

**Comment by meltsufin** [comment]
@aslo PTAL

File: scripts/cloudbuild.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@ projectRoot=`dirname $0`/..
 
 DOCKER_NAMESPACE='gcr.io/$PROJECT_ID'
 RUNTIME_NAME="openjdk"
-CANDIDATE_NAME="8-`date +%Y-%m-%d_%H_%M`"
+export CANDIDATE_NAME="8-`date +%Y-%m-%d-%H-%M`"
```
Comment (gregw) [inline scripts/cloudbuild.sh]: Why is it `CANDIDATE_NAME` rather than `LABEL`?    Other than that, this is good and builds fine for me.

Comment (aslo) [inline scripts/cloudbuild.sh]: :+1: maybe it makes sense to change this to `LONG_TAG` to match what's in the pom?

File: pom.xml [hunk 2]
```diff
@@ -28,12 +28,14 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <maven.build.timestamp.format>yyyy-MM-dd_HH_mm</maven.build.timestamp.format>
+    <maven.build.timestamp.format>yyyy-MM-dd-HH-mm</maven.build.timestamp.format>
```
Comment (aslo) [inline pom.xml]: Just curious, what's the reason for changing the underscores to hyphens here?

Comment (meltsufin) [inline pom.xml]: It's for consistency with jetty-runtime mostly.

File: scripts/cloudbuild.sh (hunk 1 above)
Comment (meltsufin) [inline scripts/cloudbuild.sh]: I'll go with `DOCKER_TAG_LONG` to match terminology in POM.

**Pull request merged**